assets/card.css

```diff
@@ -0,0 +1,9 @@
+.fixedCard {
+  @apply flex flex-wrap rounded-10;
+  @apply shadow-small bg-white relative z-card;
+  @apply flex-col overflow-hidden;
```

Comment:
I recall `overflow-hidden` on the whole card was problematic with some cases of the layout. Can we add a container with the overflow just around the image?

Reply:
why would we add a container if this works? i would be more curious why you think is not working. maybe we needed to add the rounded corners on the images due to the fact we were having a slider instead of an image or another reason, which i can't think about now. I would not add an extra wrapper just because it might break. if is breaking, sure, i would add it, but otherwise i think is not neccesarry.

Reply:
If I recall correctly it messed up the card footer in the carousel cards.

Reply:
Please check if we have it directly on the fixed card in listings. I remember spending some time on putting overflows elsewhere than in card root, but it could be that my memory is tricking me.

Reply:
we don not have it directly on the fixed card in listings. but i will check for you.

Reply:
here 
<img width="1381" alt="Screenshot 2020-08-12 at 12 48 40" src="https://user-images.githubusercontent.com/5435020/90006961-2ca05200-dc9a-11ea-9906-a832fc2fd5c1.png">

Reply:
Ah snap, nvm. It wasn't the footer. It was our initial tooltip implementation that was problematic. That's not the case anymore I think.